std_msgs/Header header
ultralytics_ros/BBox2D[] bboxes